spec/models/salary_per_year_spec.rb

```diff
@@ -25,6 +25,15 @@
       end
     end
 
+    context 'when monthly frequency' do
+      let(:salary) { 1000 }
+      let(:salary_frequency) { 'month' }
+
+      it 'returns salary multiplied by 12' do
+        expect(convert).to eq(12_000)
+      end
+    end
+
     context 'when inexistent frequency' do
       let(:salary) { 1000 }
       let(:salary_frequency) { 'inexistent' }
```

Comment:
This text `context 'when inexistent frequency' do` in L#35 will need alteration too. 

`context 'when inexistent frequency' do` --> `context 'non-existent frequency' do`